### PR TITLE
Update customer stories page

### PR DESCRIPTION
**Pull request by swillis**
**What**
Updating the Customer Stories index and individual case study pages.

**Why**
We have a new structure for our case studies, and we want to add lots more of them. We also want to let users filter case studies by business size to help them find case studies more relevant to them.

**Index Page**

screen shot 2016-08-04 at 17 37 03

**Case study page**

screen shot 2016-08-04 at 17 37 16

**To do**
- [x] Finalise copy
- [x] Add first batch of case studies
- [x] Sign off & merge

### CODE REVIEW

**Comment by clippy** [comment]
**I have manually checked the visuals in:**

- [ ] Chrome
- [ ] IE11
- [ ] Firefox
- [ ] Safari
- [ ] iPad

**Comment by swillis** [comment]
@harrison & @jamesshedden — please could you review when you get some time? Gracias 🙇

File: app/css/components/stories-index.scss [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  position: absolute;
+  bottom: 0;
+  left: 50%;
+  margin-left: -8px;
+  width: 0;
+  height: 0;
+  border-style: solid;
+  border-width: 0 8px 8px 8px;
+  border-color: transparent transparent $primary-color-xlight transparent;
+}
```
Comment (harrison) [inline app/css/components/stories-index.scss]: move this to a `industry-filters` component

File: app/css/components/stories-index.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+.stories-list {
+  background-color: $primary-color-xlight;
+}
```
Comment (harrison) [inline app/css/components/stories-index.scss]: the component file name should match the class, could rename class to stories-index or rename the component stories-list

File: app/css/components/story-cards.scss [hunk 3]
```diff
@@ -0,0 +1,85 @@
+.story-card {
```
Comment (harrison) [inline app/css/components/story-cards.scss]: rename the file `story-card`

File: app/css/components/story-page.scss [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+  width: 90%;
+  box-shadow: $drop-shadow-small;
+  margin-top: 40px;
+  margin-bottom: 40px;
+}
+
+.sidebar-logo {
+  max-width: 200px;
+  max-height: 60px;
+}
```
Comment (harrison) [inline app/css/components/story-page.scss]: could nest all these under `story-page` if they are related, e.g:
```
.story-page__image ..
.story-page__logo ..
.story-page__quote ..
```

File: app/pages/stories/stories.en.js [hunk 5]
```diff
@@ -1,148 +1,169 @@
...
+      quote: `It would have been foolish not to switch over to GoCardless`,
+      size: '120 employees',
+      tag: 'slm-connect',
+      industry: 'Web hosting',
+    },
+    {
+      link: 'stories_funding_circle',
+      quote: `GoCardless means more time to rest, which makes me a better trainer`,
+      size: '1 (just Stella)',
+      tag: 'stella-bartram',
```
Comment (harrison) [inline app/pages/stories/stories.en.js]: would maybe have the image size and the full image url here to make it easier to find and edit

```
logoWidth: 50,
logoSrc: '/images/stories/story-logos/stella-bartram.png',
```

then just set the src and img width attribute in the template

File: app/pages/stories/stories.en.js [hunk 6]
```diff
@@ -1,148 +1,169 @@
...
+      industry: 'Developers',
+    },
+  ],
+};
+
+const allStories = _.values(caseStudies).reduce((fold, value) => fold.concat(value), []);
+
+const groupedStories = _.merge({
+  All: allStories,
```
Comment (harrison) [inline app/pages/stories/stories.en.js]: could move `All` to a variable like `const DEFAULT_STORY_CATEGORY ..`

then reference it here

```
.. _.merge({
  [DEFAULT_STORY_CATEGORY]: allStories,
}..
```

File: app/pages/stories/stories.en.js [hunk 7]
```diff
@@ -1,148 +1,169 @@
...
+}, caseStudies);
 
 export default class StoriesEn extends React.Component {
   displayName = 'StoriesEn'
 
+  constructor(props) {
+    super(props);
+    this.state = { activeCategory: 'All' };
```
Comment (harrison) [inline app/pages/stories/stories.en.js]: also `DEFAULT_STORY_CATEGORY`

File: app/pages/stories/stories/funding-circle.js [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+          </h2>
+          <p className="para">
+            According to the payments team, moving to GoCardless meant a dramatic reduction in admin overheads almost immediately, with customer refunds and partial payments “much easier to process”.
+          </p>
+          <p className="para">
+            But as Funding Circle continued to  grow, it soon became apparent that GoCardless Pro – which is designed for larger firms and enables complete control over payment pages and customer notifications – would be an even better solution.
+          </p>
+          <p className="para">
+            So in March 2015 the team progressed to GoCardless Pro and took advantage of the improved API to automate more of the process. “Now with access to the API we can automate a huge number of tasks, which results in considerable time saving,” explains Jinden Badesha, Senior Product Manager at Funding Circle. “What’s more, GoCardless Pro allows us to pay out to multiple bank accounts with a single integration.”
+          </p>          <p className="para">
```
Comment (harrison) [inline app/pages/stories/stories/funding-circle.js]: indentation

File: app/pages/stories/story-page-new.js [hunk 9]
```diff
@@ -0,0 +1,104 @@
...
+
+  static propTypes = {
+    title: React.PropTypes.string.isRequired,
+    benefits: React.PropTypes.string.isRequired,
+    company: React.PropTypes.string.isRequired,
+    author: React.PropTypes.string.isRequired,
+    bgimg: React.PropTypes.string,
+    logoimg: React.PropTypes.string,
+    children: React.PropTypes.node.isRequired,
+    sidebarLogo: React.PropTypes.string.isRequired,
```
Comment (harrison) [inline app/pages/stories/story-page-new.js]: would also take the full src here, something like `sidebarLogoSrc`, having just a part of the name makes it harder to know where the image is coming from

**Comment by harrison** [comment]
💯

**Comment by Ken-Lo** [comment]
@swillis @sjnorth First six case studies have been added in so pretty close to being able to get this live! I've attached screenshots in the post above.

Would you guys mind catching up tomorrow for a final checkover of this PR? Still left on the list:

- [ ] Decide on the copy at the top of the Stories homepage
- [ ] Replace background images so they don't all have the same one!

File: app/pages/stories/stories.en.js [hunk 10]
```diff
@@ -1,146 +1,177 @@
...
+        <div className='page-hero page-hero--large page-hero--stories page-hero--stories-index u-relative u-size-full'>
           <div className='site-container page-hero__container'>
-            <div className='page-hero__inner stories-hero-heading'>
-              <div className='page-hero__text'>
-                <h1 className='u-text-heading u-color-invert u-text-center u-text-xl u-text-light'>
-                  Thousands of happy customers
+            <div className='page-hero__inner'>
+              <div className='page-hero__text u-text-center'>
+                <h1 className='u-text-heading u-color-invert u-text-xl u-text-light u-margin-Bm'>
+                  “Update Me”
```
Comment (Ken-Lo) [inline app/pages/stories/stories.en.js]: TODO: Update me and also the citation below

File: app/pages/stories/stories/boost-capital.js [hunk 11]
```diff
@@ -0,0 +1,84 @@
...
+
+  render() {
+    return (
+      <StoryPageNew
+        title='Being able to offer our customers truly paperless Direct Debit is a great selling point. It also helps to speed up on-boarding of customers.'
+        author='Alex Littner'
+        position='Managing Director'
+        company='Boost Capital'
+        benefits='Fully automated service, fast funding and a human touch.'
+        bgimgUrl='url(/images/stories/stories-hero-image.jpg)'
```
Comment (Ken-Lo) [inline app/pages/stories/stories/boost-capital.js]: Needs a new hero image

File: app/pages/stories/stories/bulb-energy.js [hunk 12]
```diff
@@ -0,0 +1,83 @@
...
+
+  render() {
+    return (
+      <StoryPageNew
+        title='GoCardless was the most simple, reliable and advanced Direct Debit system we found.'
+        author='Hayden and Amit'
+        position='Co-founders'
+        company='Bulb'
+        benefits='Low-cost, renewable energy combined with market-leading customer service.'
+        bgimgUrl='url(/images/stories/stories-hero-image.jpg)'
```
Comment (Ken-Lo) [inline app/pages/stories/stories/bulb-energy.js]: Needs a new hero image

File: app/pages/stories/stories/lyles-sutherland.js [hunk 13]
```diff
@@ -0,0 +1,129 @@
...
+  render() {
+    return (
+      <StoryPageNew
+        title='GoCardless has been a real game-changer. It’s been fantastic knowing that money is coming in every month without us
+          having to chase it.'
+        author='Fraser Sutherland'
+        position='Co-Founder'
+        company='Lyles Sutherland'
+        benefits='Removing the headache of chasing late payments and helping to make the business scalable over time.'
+        bgimgUrl='url(/images/stories/stories-hero-image.jpg)'
```
Comment (Ken-Lo) [inline app/pages/stories/stories/lyles-sutherland.js]: Needs a new hero image

File: app/pages/stories/stories/nutmeg.js [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+
+  render() {
+    return (
+      <StoryPageNew
+        title='More than 3 out of 5 of our new customers have set up direct debits.'
+        author='Bill Packman'
+        position='COO'
+        company='Nutmeg Saving and Investment'
+        benefits='Ease of use for Nutmeg’s customers, integration, easy-to-use APIs and ability to avoid the legacy bank payments network.'
+        bgimgUrl='url(/images/stories/stories-hero-image.jpg)'
```
Comment (Ken-Lo) [inline app/pages/stories/stories/nutmeg.js]: Needs a new hero image

File: app/pages/stories/stories/smart-pension.js [hunk 15]
```diff
@@ -0,0 +1,101 @@
...
+
+  render() {
+    return (
+      <StoryPageNew
+        title='GoCardless ensures our transactions are 100% accurate, easy to setup and manage for our clients. It is also significantly cheaper compared to credit card.'
+        author='Will Wynne'
+        position='Co-Founder'
+        company='Smart Pension'
+        benefits='Technology-focused approach, bolted on to a robust and secure underlying pension product.'
+        bgimgUrl='url(/images/stories/stories-hero-image.jpg)'
```
Comment (Ken-Lo) [inline app/pages/stories/stories/smart-pension.js]: Needs a new hero image

**Comment by Ken-Lo** [comment]
Going to merge in now :) Thanks @swillis @sjnorth @harrison - great work!

**Pull request merged**